invenio_rdm_records/cli.py

```diff
@@ -17,8 +17,11 @@
 from flask.cli import with_appcontext
 from flask_principal import Identity
 from invenio_access.permissions import system_identity
+from invenio_vocabularies.proxies import \
+    current_service as current_vocabularies_service
 
 from .fixtures import FixturesEngine
+from .proxies import current_rdm_records
 from .services import RDMDraftFilesService, RDMRecordService
```

Comment:
should we maybe have the other CLI commands fetch their record/draft services from the `current_rdm_records` too?
i think, that would honor custom service configurations

Reply:
Yes, that would be great......they should all use the initialised versions instead of creating it on the spot......i fixed another one like this in App-RDM today.